RELEASE.md

```diff
@@ -0,0 +1,29 @@
+# Release Process
+
+This document describes the process for releasing a new version of the `builtin-actors` project.
+
+## Current State
+
+1. Create a pull request which updates the `workspace.package.version` in the [top-level `Cargo.toml` file](https://github.com/filecoin-project/builtin-actors/blob/master/Cargo.toml).
+   - Title the PR `chore: release X.Y.Z`
+2. On pull request creation, a [Release Checker](.github/workflows/release-check.yml) workflow will run. It will perform the following actions:
+    1. Extract the version from the top-level `Cargo.toml` file.
+    2. Check if a git tag for the version already exists. Continue only if it does not.
+    3. Create a draft GitHub release with the version as the tag.
```

Comment:
I think what will help is to clarify when does the git tag get created.  My limited understanding is that a git tag is needed for creating a github release (even a draft/prerelease).  

Step 2 here says no tag should have been created yet. That sounds good and makes sense.
Step 3 seems to imply that a tag has been created at this point.  

Does the tag get created between steps 2 and 3?